regulation/changes.py

```diff
@@ -233,11 +234,20 @@ def process_changes(original_xml, notice_xml, dry=False):
 
         # Handle existing elements
         if op in ('moved', 'modified', 'deleted'):
-            # Find a match to the given label
-            matching_elm = new_xml.find('.//*[@label="{}"]'.format(label))
-            if matching_elm is None:
-                raise KeyError("Unable to find label {} to be "
-                               "{}".format(label, op))
+            # Find a match to the given label and subpath (optional)
+            if subpath is not None:
+                findstr = str('.//*[@label="{}"]/{}{}'.format(label, "{eregs}", subpath))
+                matching_elm = new_xml.find(findstr)
+                logging.debug("Performing {} operation on '{}'".format(op, findstr))
```

Comment:
Might be a small thing, but we could reduce the duplicate `logging.debug()` calls by just putting one of them below the `if`/`else`.